tests/system/action/motion/test_create.py

```diff
@@ -409,3 +409,1358 @@ def test_create_with_submitters(self) -> None:
+        assert model.get("state_id") == 34
+        assert model.get("number") == "024"
+        assert model.get("number_value") == 24
+
+    def test_create_set_number_unique_check_jump(self) -> None:
+        self.create_model(
+            "meeting/222",
+            {
+                "name": "name_SNLGsvIV",
+                "motions_number_min_digits": 3,
+                "motions_number_type": "per_category",
+            },
+        )
+        self.create_model(
+            "motion_workflow/12",
+            {"name": "name_workflow1", "first_state_id": 34, "state_ids": [34]},
+        )
+        self.create_model(
+            "motion_state/34",
+            {"name": "name_state34", "meeting_id": 222, "set_number": True},
+        )
+        self.create_model(
+            "motion_category/176", {"name": "name_category_176", "meeting_id": 222}
+        )
+        self.create_model(
+            "motion/8",
+            {
+                "title": "title_FcnPUXJB",
+                "meeting_id": 222,
+                "state_id": 34,
+                "number": "001",
+                "number_value": 23,
+                "category_id": 176,
+            },
+        )
+        self.create_model(
+            "motion/6",
+            {
+                "title": "title_FcnPUXJB",
+                "meeting_id": 222,
+                "state_id": 34,
+                "number": "024",
+                "number_value": 23,
+                "category_id": 176,
+            },
+        )
+
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "motion.create",
+                    "data": [
+                        {
+                            "title": "test_Xcdfgee",
+                            "meeting_id": 222,
+                            "workflow_id": 12,
+                            "text": "test",
+                            "category_id": 176,
+                        }
+                    ],
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        model = self.get_model("motion/9")
+        assert model.get("title") == "test_Xcdfgee"
+        assert model.get("meeting_id") == 222
+        assert model.get("state_id") == 34
+        assert model.get("number") == "025"
+        assert model.get("number_value") == 25
+
+    def _create_models_for_number_manually_tests(self) -> None:
+        self.create_model(
+            "meeting/222", {"name": "name_SNLGsvIV", "motions_number_type": "manually"}
+        )
+        self.create_model(
+            "motion_workflow/12",
+            {
+                "name": "name_workflow1",
+                "first_state_id": 34,
+                "state_ids": [34],
+            },
+        )
+        self.create_model(
+            "motion_state/34",
+            {"name": "name_state34", "meeting_id": 222, "set_number": True},
+        )
+
+    def test_issue24_manually_1(self) -> None:
```

Comment:
please choose another description than `issue24`.

Reply:
Done.

Reply:
Renamed it into "complex_example" tests.